src/nuclio/functions/version/version-configuration/tabs/version-configuration-labels/version-configuration-labels.component.js

```diff
@@ -38,8 +38,14 @@
             name: $i18next.t('common:LABEL', {lng: lng})
         });
         ctrl.validationRules = {
-            key: [],
-            value: []
+            key: ValidationService.getValidationRules('function.label.key').concat([
```

Comment:
Please use second argument of `ValidationService.getValidationRules` instead of `.concat()`